tgg/seqr_scripts/pull_seqr_pedigrees.py

```diff
@@ -21,27 +25,43 @@ def pull_project_peds(session_id: str, projects: set):
     """
     with requests.Session() as s:
         # Initialize session with authenticated cookie
-        s.cookies.set_cookie(requests.cookies.create_cookie('sessionid', session_id))
-
-        with open("seqr_pedigrees.txt", "w") as final_ped, open("projects_not_pulled.txt", "w") as errors:
+        s.cookies.set_cookie(requests.cookies.create_cookie("sessionid", session_id))
+        with open("seqr_pedigrees.txt", "w") as final_ped, open(
+            "projects_not_pulled.txt", "w"
+        ) as errors:
             final_ped.write(
                 "Project_GUID\tFamily_ID\tIndividual_ID\tPaternal_ID\tMaternal_ID\tSex\n"
             )
             for project_guid in projects:
                 project_guid = project_guid.rstrip()
-                r = s.get(
-                    f"{SEQR_URL}/api/project/{project_guid}/details"
+                family_r = s.get(
+                    f"{SEQR_URL}/api/project/{project_guid}/{GET_FAMILIES_URL}"
                 )
-
-                if r.status_code != requests.codes["ok"]:
-                    logger.info(f"Could not find {project_guid} in seqr")
+                individual_r = s.get(
+                    f"{SEQR_URL}/api/project/{project_guid}/{GET_INDIVIDUALS_URL}"
+                )
+                if (family_r.status_code != requests.codes["ok"]) or (
+                    individual_r.status_code != requests.codes["ok"]
+                ):
+                    if family_r.status_code != requests.codes["ok"]:
+                        logger.info(
+                            f"Could not get family IDs for {project_guid} in seqr."
+                        )
+                    if individual_r.status_code != requests.codes["ok"]:
+                        logger.info(
+                            f"Could not get individual IDs for {project_guid} in seqr."
+                        )
                     errors.write(f"{project_guid}\n")
                 else:
-                    fams = r.json()["familiesByGuid"]
+                    # family_r is stuctured with a high level field familiesByGuid, which contains fields for individual families, which themselves contain various fields, like analysisStatus, analyzedBy, etc.
+                    # importantly, familiesByGuid has fields familyGuid, which is a more keyword type family name, and familyId, which is the family name used in seqr
+                    fams = family_r.json()["familiesByGuid"]
                     fams_dict = {}
                     for fam in fams.values():
                         fams_dict[fam["familyGuid"]] = fam["familyId"]
-                    inds = r.json()["individualsByGuid"]
+                    # individual_r is structured with high level field individualsByGuid which contain fields for individuals, such as paternalId, maternalId, and familyGuid. Critically it does not contain
+                    # familyId, which is why we need to build a mapping from familyGuid to familyId.
```

Comment:
```suggestion
                    # `individual_r` contains the high level field `individualsByGuid`, which contains fields for individuals
                    # such as `paternalId`, `maternalId`, and `familyGuid`. 
                    # Critically, it does not contain `familyId`, 
                    # which is why we need to build a mapping from `familyGuid` to `familyId`.
```
(optional, slight rewording/reformatting)